docs/station.md

```diff
@@ -0,0 +1,49 @@
+# station (Private Beta)
+A hybrid station has the capability to trigger a scan within your Virtual Private Cloud (VPC) environment based on the inputs provided through the HCP UI.
+The `station` command is used to fetch scan jobs from [HCP](https://portal.cloud.hashicorp.com/)and execute them in your VPC. It will scan any available jobs, otherwise it will sleep and wait for a job to be queued in HCP.
+
+## Setting Up
+In order to run `station` in your environment there are 3 configuration steps required in HCP and your local environment.
+
+### Create a Service Principal
+**Note** Does the type of SP matter in this case?
```

Comment:
Does this need to be a project level SP? Or will org level SP work?

Reply:
either works, but let's recommend project level